nimble/host/audio/include/audio/ble_audio.h

```diff
@@ -536,17 +536,8 @@ struct ble_audio_broadcast_name {
 /** BLE Audio event: BASS - Remote Scan Started */
 #define BLE_AUDIO_EVENT_BASS_REMOTE_SCAN_STARTED             4
 
-/** BLE Audio event: BASS - Add Source */
```

Comment:
What's the reason to remove these events? I guess the upper layer knows that some actions were performed because the action callback will be called, but this callback is not mandatory. So if the operation was accepted autonomously, there will be no feedback for upper layer/app. For example, we could never enter here:
https://github.com/apache/mynewt-nimble/pull/1677/commits/e5a346bdc39590f58896fdbf25262088cde34480#diff-b8c009ce2010b2473424d883cd93e8fe365b1c1af565ad8ba875280fd069133cR324

Reply:
not applicable